config/crd/bases/compliance.openshift.io_tailoredprofiles.yaml

```diff
@@ -97,13 +107,18 @@ spec:
                   description: RuleReferenceSpec specifies a rule to be selected/deselected,
                     as well as the reason why
                   properties:
+                    kind:
+                      description: Type of rule, either "Rule" or "CustomRule". "Rule"
+                        is the default if not specified.
+                      type: string
                     name:
                       description: Name of the rule that's being referenced
                       type: string
                     rationale:
                       description: Rationale of why this rule is being selected/deselected
                       type: string
                   required:
+                  - kind
```

Comment:
Similar comment here as above. Does it makes sense to support `CustomRules` from a manual rule perspective in a tailored profile?

Reply:
I can write a check later in tailoredProfile controller to check for that